index.md

```diff
@@ -14,23 +14,25 @@ See the SIG constitution for more information. Membership is free of charge and
 
 To join, just register to our [Google group](https://groups.google.com/forum/#!forum/rosp-sig).
 
-You are also welcome to join our [LinkeIn](https://www.linkedin.com/groups/6744473/) group
+You are also welcome to join our [LinkedIn](https://www.linkedin.com/groups/6744473/) group
 
-# [Datasets](https://isca-sig-rosp.github.io/ISCA-SIG-RoSP/datasets.html)
+# [Datasets](/datasets.md)
 
 This page provides a list of datasets, with corresponding software baselines and evaluation results.
 
 # [Software](https://wiki.inria.fr/rosp/Software)
 This page provides a list of speech processing software and corresponding application recipes for different datasets.
 
 # SIG events
```

Comment:
Should we also create a google sheet for events?
That may be easier to maintain?

Reply:
I'll add a TODO for this for the future. In the meantime, I suppose we can merge these changes for now.